application/src/main/resources/thingsboard.yml

```diff
@@ -1411,6 +1411,8 @@ queue:
     max_msg_size: "${TB_QUEUE_PUBSUB_MAX_MSG_SIZE:1048576}"
     # Number of messages per consumer
     max_messages: "${TB_QUEUE_PUBSUB_MAX_MESSAGES:1000}"
+    # Thread pool size for pubsub queue executor provider. If not set - default pubsub executor provider value will be used (5 * number of available processors)
```

Comment:
@dashevchenko , @smatvienko-tb ,  why 5xCPU count? Is this a recommended value?

Reply:
this is default thread pool size value for default provider